W-16026636 Making fixes (#341)
mulesoft/docs-access-management: modules/ROOT/pages/conf-ldap-private-cloud-task.adoc

= Configuring LDAP in Anypoint Platform Private Cloud
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Configure user management using LDAP version 3 for the Private Cloud Edition of Anypoint Platform. User management through LDAP is available only for the Private Cloud Edition.

Reviewer: ```suggestion
Configure user management using LDAP version 3 for Anypoint Platform Private Cloud Edition (Anypoint PCE). User management through LDAP is available only for Anypoint PCE.
```
